src/evaluate.py

```diff
@@ -123,6 +132,10 @@ def evaluate(
 
         # Combine task_preds from each batch to a single DataFrame.
         task_preds = pd.concat(task_preds, ignore_index=True)
+        if task.name.startswith("winogender"):
```

Comment:
This seems like an odd place for a special case. If this really needs to be here, add a comment to the code saying why.